package.json

```
@@ -4,7 +4,7 @@
   "main": "index.js",
   "repository": "git@github.com:discourse/discourse.git",
   "author": "Discourse",
-  "license": "MIT",
+  "license": "GPLv2",
```

I noticed discourse had MIT in our package.json so I took the opportunity to correct it.

---

I see it generates a warning:

> warning package.json: License should be a valid SPDX license expression

Looking at the list (https://spdx.org/licenses/), it apparently should be:

```suggestion
  "license": "GPL-2.0-only",
```

(or `GPL-2.0-or-later`)

---

Weird I literally got that string from a list of valid values! I guess it was wrong or outdated.